@W-23154741 - Devops promote commands (#510)
salesforcecli/plugin-devops-center: messages/devops.work-item.prepare.md

# summary

Prepare a work item for one-off promotion between pipeline stages.

# description

Prepares a work item for one-off promotion by creating the necessary branches and pull requests in the source control repository. Run this command before running "sf devops work-item promote" for a one-off promotion.

Use "sf devops work-item list" to find the work item ID and target stage ID for a project.

# flags.work-item-id.summary

ID of the work item to prepare for one-off promotion.

# flags.target-stage-id.summary

ID of the pipeline stage to promote the work item to.

# examples

> - Prepare a work item for one-off promotion to the UAT stage:

Reviewer: ```suggestion
- Prepare a work item for one-off promotion to the UAT stage; use the org with alias "my-devops-org":
```

      <%= config.bin %> <%= command.id %> --target-org my-devops-org --work-item-id 0Wx000000000001 --target-stage-id 05S000000000002

> - Prepare a work item using a DevOps Center org username:

Reviewer: ```suggestion
- Prepare a work item; use an org with the specified username:
```
